.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "RoadNamer/Libraries/Cimtools"]
+	path = RoadNamer/Libraries/Cimtools
+	url = git@github.com:tony56a/CimTools.git
```

Comment:
Whoops, almost forgot, this needed to be changed back to your repo once PropaneDragon/Cimtools#4 merges